tests/rptest/tests/compatibility/sarama_scram_test.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import os
+import subprocess
+import string
+from ducktape.mark.resource import cluster
+
+from rptest.clients.types import TopicSpec
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.python_librdkafka import PythonLibrdkafka
+
+#Compatibility testing for Sarama
+class SaramaScramTest(RedpandaTest):
+
+    CLIENT_DIR = "/kafka-clients"
+    TESTS_DIR = os.path.join(CLIENT_DIR, "sarama")
+
+    def __init__(self, test_context):
+        extra_rp_conf = dict(
+            developer_mode=True,
+            enable_sasl=True,
+        )
+        super(SaramaScramTest, self).__init__(test_context,
+                                        extra_rp_conf=extra_rp_conf)
+
+        self.sarama_topic = "sarsecure"
+
+        # Check if CLIENT Dir exists first
+        try:
+            os.mkdir(SaramaScramTest.CLIENT_DIR)
+        except FileExistsError:
+            self.logger.debug("Not creating {}. It already exists".format(SaramaScramTest.CLIENT_DIR))
+
+        # Check if sarama dir already exists first
+        if os.path.isdir(SaramaScramTest.TESTS_DIR):
+            self.logger.debug("Not downloading {}. It already exists".format(SaramaScramTest.TESTS_DIR))
+        else:
+            self._execute("git clone https://github.com/Shopify/sarama.git", SaramaScramTest.CLIENT_DIR)
```

Comment:
I see your point. I will put dependencies in the dockerfile

Reply:
yes, putting them in the dockerfile is desirable, so that this gets cached and doesn't get installed on every run. this will only be used in ducktape tests, right? Another alternative would be to automate this with Task and install to `vbuild/`, but that would only be needed if these clients are used outside tests.

Reply:
thanks for chiming in. I think right now it makes sense to stick them in the docker file.